### PR TITLE
Lets see if this fixes yahoos inability to respond to a legit query

### PR DESCRIPTION
**Pull request by CrazyLemon**

### CODE REVIEW

File: scripts/vreme.coffee [hunk 1]
```diff
@@ -67,10 +67,13 @@ module.exports = (bot) ->
           id = res.place.woeid
 
         yql "select item from weather.forecast where woeid = \"" + id + "\"", (res) ->
-          item = res.channel.item
-          cb "#{(100 / (212 - 32) * (item.condition.temp - 32)).toFixed(2)}°C #{item.link}"
+          if res?
+            item = res.channel.item
+            cb "#{(100 / (212 - 32) * (item.condition.temp - 32)).toFixed(2)}°C #{item.link}"
+          else
+            cb "Podatka o vremenu ni ... ker yahoo sucks"
```
Comment (zdobersek) [inline scripts/vreme.coffee]: Yahoo*

File: scripts/vreme.coffee [hunk 2]
```diff
@@ -67,10 +67,13 @@ module.exports = (bot) ->
           id = res.place.woeid
 
         yql "select item from weather.forecast where woeid = \"" + id + "\"", (res) ->
-          item = res.channel.item
-          cb "#{(100 / (212 - 32) * (item.condition.temp - 32)).toFixed(2)}°C #{item.link}"
+          if res?
+            item = res.channel.item
+            cb "#{(100 / (212 - 32) * (item.condition.temp - 32)).toFixed(2)}°C #{item.link}"
```
Comment (zdobersek) [inline scripts/vreme.coffee]: 212 - 32 = 180.

Comment (CrazyLemon) [inline scripts/vreme.coffee]: You so smart! Thats why i assigned you!

**Comment by zdobersek** [comment]
Oh, lih sem se spomnu -- meh mi je.

**Pull request merged**